src/biophysics.jl

```diff
@@ -284,8 +284,35 @@ function convection(Body, area, T_air, T_surf, vel, P_atmos, elev, fluid)
     (Q_conv=Q_conv, Hc=Hc, Hd=Hd, Sh=Sh, Q_free=Q_free, Hc_free=Hc_free, Hc_forc=Hc_forc, Sh_free=Sh_free, Sh_forc=Sh_forc, Hd_free=Hd_free, Hd_forc=Hd_forc)
 end
 
-function conduction(A, L, T_org, T_sub, k_sub)
-    A * (k_sub / L) * (T_org - T_sub)
+convection(Body, p::Model, o::OrganismalVars, e::EnvironmentalVars) = begin
+    model_pars = stripparams(p)
+    env_pars = model_pars[2]
+    Ac = Body.geometry.area * (1 - o.p_cond)
+    convection(Body, Ac, e.Ta, o.T_surf, e.vel, env_pars.P_atmos, env_pars.elev, env_pars.fluid)
+end
+
+convection(T_x, Body, p::Model, o::OrganismalVars, e::EnvironmentalVars) = begin
+    model_pars = stripparams(p)
+    env_pars = model_pars[2]
+    Ac = Body.geometry.area * (1 - o.p_cond)
+    convection(Body, Ac, e.Ta, T_x, e.vel, env_pars.P_atmos, env_pars.elev, env_pars.fluid)
+end
+
+
+function conduction(A, L, T_surf, T_sub, k_sub)
+    A * (k_sub / L) * (T_surf - T_sub)
+end
+
+conduction(Body, p::Model, o::OrganismalVars, e::EnvironmentalVars) = begin
+    L = 0.025m
+    Av = Body.geometry.area * o.p_cond
+    conduction(Av, L, o.T_surf, e.Tsub, e.k_sub)
+end
+
+conduction(T_x, Body, p::Model, o::OrganismalVars, e::EnvironmentalVars) = begin
+    L = 0.025m
```

Comment:
This function has an identical signature to the one further above?

Maybe you mean to be dispatching on the type of `Body` and the contents of `Model`?

Reply:
No worries, think I actually misunderstood all of these methods from the close names of `convection` and `conduction`. 

To avoid this, when  the method is for the same function (like `conduction`) we often remove the empty lines between them to show the groupings more clearly.

You may just want to add some types to this, like `::Number` as currently this is a fallback for everything with 5 arguments:

`function conduction(A, L, T_surf, T_sub, k_sub)`